### ISSUE TITLE
Webpack error

### ISSUE THREAD

**Issue by sxwebdev** [xaviergonz/mobx-keystone#235]
I try to reproduce the todo list example and get an webpack error.

Sample code below.

```ts
import React from "react";
import { computed } from "mobx";
import {
  model,
  Model,
  modelAction,
  ModelAutoTypeCheckingMode,
  registerRootStore,
  setGlobalConfig,
  //tProp,
  prop,
  //types,
} from "mobx-keystone";

setGlobalConfig({
  modelAutoTypeChecking: ModelAutoTypeCheckingMode.AlwaysOn,
});

@model("nv/Todo")
export class Todo extends Model({
  //   id: tProp(types.string, () => uuidv4()),
  //   text: tProp(types.string),
  //   done: tProp(types.boolean, false),

  id: prop<string | undefined>(),
  text: prop<string>(),
  done: prop<boolean>(false),
}) {
  @modelAction
  setDone(done: boolean): void {
    this.done = done;
  }

  @modelAction
  setText(text: string): void {
    this.text = text;
  }
}

@model("nv/TodoList")
export class TodoList extends Model({
  //todos: tProp(types.array(types.model(Todo)), () => []),
  todos: prop<Todo[]>(() => []),
}) {
  @computed
  get pending(): Todo[] {
    return this.todos.filter((t) => !t.done);
  }

  @computed
  get done(): Todo[] {
    return this.todos.filter((t) => t.done);
  }

  @modelAction
  add(todo: Todo): void {
    this.todos.push(todo);
  }

  @modelAction
  remove(todo: Todo): void {
    const index = this.todos.indexOf(todo);
    if (index >= 0) {
      this.todos.splice(index, 1);
    }
  }
}

export function createRootStore(): TodoList {
  const rootStore = new TodoList({
    todos: [
      new Todo({ text: "make mobx-keystone awesome!" }),
      new Todo({ text: "spread the word" }),
      new Todo({ text: "buy some milk", done: true }),
    ],
  });

  registerRootStore(rootStore);

  return rootStore;
}

@model("nv/RootStore")
class RootStore extends Model({ todoList: prop<TodoList>() }) {
  @modelAction
  setTodos(userPreferences: TodoList): void {
    this.todoList = userPreferences;
  }
}

export const myRootStore = new RootStore({ todoList: new TodoList({}) });

registerRootStore(myRootStore);

const StoreContext2 = React.createContext<RootStore>({} as RootStore);

export const useStoreKS: () => RootStore = () =>
  React.useContext(StoreContext2) as RootStore;
export const StoreProvider2 = StoreContext2.Provider;

export type { RootStore };
```

Error:

![image](https://user-images.githubusercontent.com/3129246/115841026-06d06d00-a425-11eb-87cf-6cc0b4764900.png)

```
"Module not found: Error: Can't resolve 'fast-deep-equal/es6' in '/Users/uname/p/www/n/node_modules/mobx-keystone/dist'
Did you mean 'index.js'?
BREAKING CHANGE: The request 'fast-deep-equal/es6' failed to resolve only because it was resolved as fully specified
(probably because the origin is a '*.mjs' file or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request."
```

I tried to install module `"fast-deep-equal": "^3.1.3"` but it doesn't help

Have any ideas what could be wrong?

**Comment by xaviergonz** [comment]
Thanks for the bug report! which webpack version are you using?

**Comment by sxwebdev** [comment]
> Thanks for the bug report! which webpack version are you using?

Thanks for the quick response

`"webpack": "^5.35.0"`

**Comment by xaviergonz** [comment]
Could you also post here the package.json so I can reproduce it locally?

**Comment by sxwebdev** [comment]
Sure. I have uploaded a test example to [this repository ](https://github.com/sxwebdev/mobx-keystone-test)

**Comment by xaviergonz** [comment]
Thanks, it should be solved in 0.56.1

**Comment by sxwebdev** [comment]
Thank you. Now works as expected

**Comment by sxwebdev** [comment]
![image](https://user-images.githubusercontent.com/3129246/115915194-ad475d00-a47b-11eb-8336-df2ef550928a.png)

For some reason now the linter started to swear

**Comment by xaviergonz** [comment]
is strict: true set on your tsconfig.json?

**Comment by xaviergonz** [comment]
if that fails try to restart the typescript service

**Comment by sxwebdev** [comment]
> is strict: true set on your tsconfig.json?

Yes. tsconfig looks [like this](https://github.com/sxwebdev/mobx-keystone-test/blob/master/configs/tsconfig-base.json)


> if that fails try to restart the typescript service

I tried this too, it didn't help

**Comment by sxwebdev** [comment]
for example, all properties for this model are not available. only actions

![image](https://user-images.githubusercontent.com/3129246/115931450-c78d3500-a493-11eb-9c85-6b0589dce145.png)

however, if you use this symbol `$` , then all properties are displayed.

![image](https://user-images.githubusercontent.com/3129246/115931493-de338c00-a493-11eb-9b07-b4b9f65cbcc1.png)

**Comment by xaviergonz** [comment]
Thanks! Should be fixed in v0.56.2 (apparently that bug got introduced in 0.55.0 :( )

**Comment by sxwebdev** [comment]
> Thanks! Should be fixed in v0.56.2 (apparently that bug got introduced in 0.55.0 :( )

Excellent! Now the error is gone! Thank you

![image](https://user-images.githubusercontent.com/3129246/115948210-74d46d00-a4d5-11eb-9cdf-77cd35335a91.png)

**Comment by airtonix** [comment]
![image](https://user-images.githubusercontent.com/61225/129475981-e279ea7a-b6ae-44cb-93d7-a786df8ca0a6.png)
